community/io/src/main/java/org/neo4j/io/fs/DefaultFileSystemAbstraction.java

```diff
@@ -133,9 +134,9 @@ public void deleteRecursively( File directory ) throws IOException
     }
 
     @Override
-    public boolean renameFile( File from, File to ) throws IOException
+    public void renameFile( File from, File to, CopyOption... copyOptions ) throws IOException
     {
-        return FileUtils.renameFile( from, to );
+        FileUtils.renameFile( from, to, copyOptions );
```

Comment:
Why we using FileUtils.renameFile here but use Files.move in DelegateFileSystemAbstraction  shouldn't we use the same thing in both?